packages/core/src/App/Components/Layout/Header/toggle-notifications.jsx

```diff
@@ -11,6 +13,22 @@ const ToggleNotificationsDrawer = ({
     tooltip_message,
     should_disable_pointer_events = false,
 }) => {
+    const {
+        notifications: { is_notifications_loading_completed },
+    } = useStore();
+
+    const [counter, setCounter] = useState(null);
+
+    // don't hide count when 'is_notifications_loading_completed' = false
+    useEffect(() => {
```

Comment:
@sergei-deriv Could we handle it in the `NotificationStore` and avoid `this.notifications` getting set if we are switching between accounts? 🤔

Reply:
@farzin-deriv Unfortunately, I'm almost sure it's impossible, because `count` is `notification.length` from `notification-store` and this count changes every time when the app filters notifications and without useEffect() toggle-notifications component will be rerendered every time. I have to think more but right now I don't have any idea how to do this without useEffect

Reply:
@farzin-deriv right now if I just left

Reply:
@farzin-deriv first time I did so

Reply:
@sergei-deriv please don't set the entire component into the local state here, please use a boolean flag for conditional rendering like it was done previously. conditional rendering should work in the same but cleaner way. as i mentioned in another comment, please avoid using negative conditions like '_don't hide count when 'is_notifications_loading_completed' = false_' for readability. Actually comments here are pretty obvious, you can omit them.  )
  Component will update every time props are updated, useEffect is not stopping it from updating, we can decide when we want to execute the code inside useEffect based on component props changes.
  This component is directly observing the is_notifications_loading_completed computed, and I suggest that we make it an observable because computed are recalculated much more often than observables are.👍🏻

Reply:
@maryia-deriv I understand useEffect is not stopping component from updating, but it allows me to save previous count and don't change it during rerender. About `computed` and `observable` I've answered below and I tried to make it observable last time, but it didn't work for me. But ok, I'll try to do this again